src/main/java/org/folio/circulation/support/http/client/CqlQuery.java

```diff
@@ -46,6 +46,12 @@
     return Result.of(() -> new CqlQuery(format("%s==\"%s\"", index, value), none()));
   }
 
+  public static Result<CqlQuery> exactMatchWithinArray(String arrayName, String key, String value) {
+    final String query = format("%s==\"*\\\"%s\\\": \\\"%s\\\"*\"", arrayName, key, value);
```

Comment:
Hi @marc-johnson. Is it even possible to make a query like this one work in API tests? Currently, this query executes nicely on a live running module, but returns nothing during tests.

Reply:
@OleksandrVidinieiev It likely is not possible to support that query in the API tests, as the fake modules used likely don't know how to interpret this CQL.

Are you asking because the tests below are failing?

Reply:
> Are you asking because the tests below are failing?

Yes. I'm trying to filter `Owners` by owned service point on DB side, mainly to avoid fetching all of them and then filter in java. Do you have any suggestions?

Reply:
It is likely that the fake CQL interpreter will need to be extended to support this kind of query.

(Alas, this is one of the challenges with using a complex, open ended query language for cross module integration)

Reply:
@marc-johnson, thanks for your feedback. Since we need this fix on Bugfest ASAP, I went the easy way.